tests/test_helpers.py

```diff
@@ -84,6 +80,21 @@
      "but with the addition of -n followed by a number corresponding to one of the "
      "above listed projectum ids that you would like to update.\n"
      ),
+    (["helper", "u_contact", "afriend", "--number", "1",
+      "--notes", "Test note", "-a", "Friend", "--date", "2020-01-02"],
+     "afriend has been added/updated in contacts\n"
+     ),
+    (["helper", "u_contact", "Anthony", "--date", "2020-01-02"],
+     "Please choose from one of the following to update/add:\n"
+     "0. Anthony as a new contact\n"
+     "1. Anthony B Friend    id: afriend\n"
+     "\n"
+     ),
```

Comment:
I am annoyed at this last \n, but spent more time than I should trying to get rid of it. 
do you have any solution?